### ISSUE TITLE
Midterm Peer Review

### ISSUE THREAD

**Issue by fort745** [Ay1man2/ORIE-4741-PROJ#6]
**Summary**

The goal of the project is to predict the number of shootings in a year and what might affect it.

**What I liked**

1. You did a good job preprocessing the data from its original form to a usable state. 
2. You are trying to tackle an important issue, and the choice to find a new dataset with both fatal and nonfatal incidents makes a lot of sense.
3. I liked how you mentioned shuffling/randomizing your training/testing samples, which is a key part of getting accurate, representative metrics.

**Areas of Improvement**

1. Could have explained more on why you chose to fit using polynomial regression as opposed to other models. Would have been nice to see some other metrics of accuracy to more easily compare models as well.
2. It seems you mentioned splitting data into training and validation sets, but forgot about a testing set.
3. Your data set seems quite small and prone to overfitting - it’s probably a better idea to impute than drop the missing values in order for you to retain enough data to make decent predictions.
4. Unsure how valuable it is to try to predict the number of shootings in a time frame. It seems like your choice of dataset would point to trying to classify whether an incident is fatal or nonfatal based on demographic, location, subject info, officer info, and other data instead.
